api/routes/user.js

```diff
@@ -107,7 +107,7 @@ router.post('/login', function (req, res) {
           if (isMatch && !error) {
             if (
               new Date() - user.membershipValidUntil > 0 &&
-              user.accessLevel < 2
+              user.accessLevel < 0
```

Comment:
no more hardcoded numbers. [we have enums for this](https://github.com/SCE-Development/Core-v4/blob/3c4ba959ba6a462c28bae5841594edf1ed625033/src/Enums.js#L28) so import them